src/backend/utils/cache/plancache.c

```diff
@@ -213,6 +213,8 @@ CreateCachedPlan(Node *raw_parse_tree,
 	plansource->generic_cost = -1;
 	plansource->total_custom_cost = 0;
 	plansource->num_custom_plans = 0;
+	plansource->init_plangen_used = PLANGEN_PLANNER;
```

Comment:
Maybe we should use `NULL` as init values?

Reply:
To use `NULL` pointer here we obviously should point to something, because we can't point to `enum` member. I don't think it's safe and clear to point to statement's `planGen`.